sys/include/net/ng_nettype.h

```diff
@@ -58,6 +58,9 @@ typedef enum {
 #ifdef MODULE_NG_ICMPV6
     NG_NETTYPE_ICMPV6,          /**< Protocol is ICMPv6 */
 #endif
+#ifdef MODULE_NG_RPL
+    NG_NETTYPE_RPL_CTRL,        /**< Protocol is ICMPv6, RPL sub-type */
```

Comment:
Demux is done in https://github.com/RIOT-OS/RIOT/pull/3050/files#diff-a263cf517da28750259527ba0aab62dfR122 so everything is as it should be :-)